packages/admin-ui/src/components/Filter/multiple-select-content.tsx

```diff
@@ -0,0 +1,66 @@
+import type { ReactNode } from 'react'
+import React, { useRef } from 'react'
+
+import type {
+  FilterItem,
+  UseMultipleFilterReturn,
+} from './useMultipleFilterState'
+
+import { Checkbox } from '../Checkbox'
+import { useOption } from '@react-aria/listbox'
+import { tag } from '@vtex/admin-ui-react'
+import { focusVisible } from '@vtex/admin-ui-core'
+import { Set } from '../Set'
+import type { ListState } from '@react-stately/list'
+
+export function MultipleSelectContent(props: FilterCheckboxProps) {
+  const {
+    state: { listBoxProps, listState, ref },
+  } = props
+
+  return (
+    <Set
+      as="ul"
+      spacing={5}
+      orientation="vertical"
+      ref={ref}
+      csx={{ margin: '$l', paddingY: '$m' }}
+      {...listBoxProps}
+    >
+      {[...listState.collection].map((item) => {
+        return <Option key={item.key} item={item} state={listState} />
+      })}
+    </Set>
+  )
+}
+
+function Option({
```

Comment:
suggestion: I would probably create a file Option.tsx than let it inside the multiple-select-content.tsx file. I think it can improve the readability of the componentes